### PR TITLE
Initial addition of python application

### PR DESCRIPTION
**Pull request by bwockner**
Example python viewer; some documentation still needs updating and some code housekeeping required

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+Date:                  2020-04-30
+Vault SDK Version:     0.5.2
+Toolsets:              Python3 Pillow Pyglet Numpy IPython3
+```
+
+## Resources Required
+<!-- TODO: Fill this section below with the resources required to do this sample-->
+
+## How to use the sample
+<!-- TODO: Explain how this sample can be used and what is required to get it running -->
```
Comment (ucpasas) [inline README.md:59]: Update to add prerequisite steps to get Vault SDK working

File: pygletExample.py [hunk 2]
```diff
@@ -0,0 +1,469 @@
+import math
+import sys
+import threading
+
+import IPython
```
Comment (ucpasas) [inline pygletExample.py:5]: Ipython will need proper referencing due to nature of license

File: pygletExample.py [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+import numpy
```
Comment (ucpasas) [inline pygletExample.py:6]: Numpy will need proper referencing due to nature of license

File: pygletExample.py [hunk 4]
```diff
@@ -0,0 +1,469 @@
...
+from PIL import Image
+
+import vault
+from easyrenderer import VDKEasyRenderer
+import pyglet.window.key as keyboard
+import pyglet
```
Comment (ucpasas) [inline pygletExample.py:12]: Pyglet will need proper referencing due to nature of license

File: README.md (hunk 1 above)
Comment (ucpasas) [inline README.md:59]: Include prerequisite steps to get Vault SDK working

File: README.md [hunk 5]
```diff
@@ -1,2 +1,75 @@
...
+and pyglet libraries in addition to Vault and Pillow
+
+
+#### pygletexample.py
+A basic python client for viewing UDS models made using pyglet (an openGL wrapper for python). It makes use of the EasyRender object interface
+`App` represents the window context that the application runs in. It handles all user input and manages dispaches the draw commands to the
+various UI objects.
+`VDKViewPort` and it's derived objects represents an OpenGL quad which is to be textured using the frame buffer written to by each call of vdkRenderContext.Render
+
+When this file is run as a python script (i.e. by running `python3 pygletexample.py` with the appropriate python environment set up, the calling
```
Comment (ucpasas) [inline README.md:99]: As per testing, will work only with "ipython3"

**Review by ucpasas — CHANGES_REQUESTED** [review]

File: pygletExample.py (hunk 2 above)
Comment (bwockner) [reply pygletExample.py:5]: referencing through the pip install should be sufficient as we are only calling these libraries

**Review by ucpasas — APPROVED** [review]

**Pull request merged**